manta-crypto/src/eclair/bool.rs

```diff
@@ -42,6 +42,13 @@ pub trait Assert: Has<bool> {
     }
 }
 
+impl Assert for () {
+    #[inline]
+    fn assert(&mut self, bit: &bool) {
+        assert!(bit)
+    }
+}
```

Comment:
When will we use `
let tmp = ();
tmp.assert(true);
`?

Reply:
Remember that `()` is the native compiler so it needs to support as many functions as possible.